docs/docs/data-storage-redux.md

```diff
@@ -1,9 +1,55 @@
 ---
 title: Data Storage (Redux)
-issue: https://github.com/gatsbyjs/gatsby/issues/21068
 ---
 
-This is a stub. Help our community expand it.
+During Gatsby's bootstrap & build phases, the state is stored and manipulated using the [Redux](https://redux.js.org/) library. The key purpose of using Redux is to centralize all of the state logic. Reviewing the Gatsby [reducers](https://github.com/gatsbyjs/gatsby/tree/80acb8d5d67f7e277ce44158b36da84d262e5b23/packages/gatsby/src/redux/reducers) and [actions](https://github.com/gatsbyjs/gatsby/tree/80acb8d5d67f7e277ce44158b36da84d262e5b23/packages/gatsby/src/redux/actions) folders gives a comprehensive picture of what state manipulations are possible.
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+## Store
+
+The namespaces in Gatsby's Redux store are a great overview of the Gatsby internals, here are a few:
+
+- [Pages](https://www.gatsbyjs.org/docs/gatsby-internals-terminology/#redux-pages-namespace)
+- [Components](https://www.gatsbyjs.org/docs/gatsby-internals-terminology/#redux-components-namespace)
+- Nodes
```

Comment:
I wish there was more info for each of these store items, as they are really interesting parts of Gatsby's architecture. I feel like general descriptions of these would be a great outcome for this doc. If it's possible to describe them in original text here, that would be ideal; but we can also open issue(s) to add docs descriptions for them if necessary.

Reply:
Oof I enjoyed looking into these, really interesting. Hopefully this is roughly what you had in mind. Tried to keep them as short as possible, they could be more consistent in how I've structured each sentence maybe?